src/metabase/models/pulse.clj

```diff
@@ -192,7 +193,7 @@
   "Update an existing `Pulse`, including all associated data such as: `PulseCards`, `PulseChannels`, and `PulseChannelRecipients`.
 
    Returns the updated `Pulse` or throws an Exception."
-  [{:keys [id name cards channels] :as pulse}]
+  [{:keys [id name cards channels skip_if_empty?] :as pulse}]
```

Comment:
The rule is if it comes back from the API or the DB than it can be `snake_case` but if it's being passed from one Clojure function to the next than it should have a proper `lisp-case` name